style.css

```diff
@@ -0,0 +1,390 @@
+}
+
+.status-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
+.status-wrapper::before {
+  z-index: 1;
+  content: '';
+  position: absolute;
+  width: 120%;
+  height: 23px;
+  background-color: blue;
+  top: -10px;
+  margin: 0 auto;
+  border-radius: 23px;
+}
+
+.status::before {
+  content: '';
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  background-color: white;
+  border-radius: 34px;
+  top: -38px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  z-index: 1;
+}
+
+.status {
+  background: #191c26;
+  border-radius: 14px;
+  margin: 25px 10px;
+  padding: 10px;
+  font-family: NexaLight;
+  text-align: left;
+  font-size: 18px;
+  font-weight: 300;
+  line-height: 30px;
+  letter-spacing: normal;
+  color: grey;
+  position: relative;
+  border: 6px solid #191c26;
+}
+
+.date {
+  font-family: NexaBold;
+}
+
+.roadmap-description {
+  font-size: 16px;
+  line-height: 28px;
+  margin: 0 0 20px;
+  font-weight: 300;
+  font-family: NexaLight;
+  color: #acb0b9;
+}
+
+#hides {
+  display: none;
+}
+
+@media (min-width: 724px) and (max-width: 724px) {
+  .backgroundImg {
+    max-width: 190px;
+    height: 190px;
+    left: 450px;
+    right: -100px;
+    top: 130px;
+    z-index: -1;
+    position: absolute;
+  }
+}
+
+@media (max-width: 667px) and (min-width: 375px) {
+  .backgroundImg {
+    max-width: 190px;
+    max-height: 190px;
+    left: 600px;
+    right: -100px;
+    top: 170px;
+    z-index: -1;
+    position: absolute;
+  }
+}
+
+@media (max-width: 991px) and (min-width: 991px) {
+  .backgroundImg {
+    max-width: 190px;
+    height: 190px;
+    left: 150px;
+    right: -100px;
+    top: 130px;
+    z-index: -1;
+    position: absolute;
+  }
+  .heading {
+    font-size: 25px;
+    line-height: 32px;
+    text-align: center;
+    position: sticky;
+  }
+  .swiper-wrapper {
+    overflow-x: scroll;
+    justify-content: flex-start;
+  }
+  .roadmap-swiper {
+    min-width: 200px;
+  }
+  .timestamp {
+    display: none;
+  }
+  .status-wrapper {
+    padding: 10px 5px;
+    position: relative;
+  }
+  .status-wrapper::before {
+    height: 13px;
+    content: '';
+    position: absolute;
+    width: 115%;
+    background-color: blue;
+    border-radius: 43px;
+    top: -1px;
+  }
+  .status {
+    max-width: 270px;
+    width: 100%;
+    border-radius: 14px;
+    margin: 5px;
+    padding: 20px;
+    text-align: left;
+    position: relative;
+    top: 15px;
+    font-size: 18px;
+    font-weight: 300;
+    line-height: 30px;
+  }
+  .status::before {
+    left: 36px;
+    margin: 0;
+    max-width: 8px;
+    max-height: 8px;
+    background-color: white;
+    border-radius: 25px;
+    border: none;
+    top: -34px;
+  }
+  .date#hides {
+    max-width: 90px;
+    width: 100%;
+    display: block;
+    background: turquoise;
+    padding: 8px 10px 6px;
+    margin: 0 0 15px;
+    box-shadow: inset 0 2px 2px #0000008a;
```

Comment:
what is inset mean can ypu please explain this line @aeshapatel025

Reply:
shadow will be drawn inside the element.